### DISCUSSION TITLE
Can't add photos with Python

### DISCUSSION THREAD

**Discussion by mattgoliver** [exadel-inc/CompreFace#408] [Q&A]
I'll start off with the fact that I'm not very experienced with coding so I had no idea how to use REST API. I did some basic research and tried using python to send a test image to the collection I set up on my computer. I got the GET method to work, but of course it didn't show any subjects since I hadn't uploaded any photos yet, the main point being it went through without any errors. On the other hand, I just can’t get the POST method to work and I was hoping someone would have some advice for me. Whenever I try sending a picture (under 5 Mb) it returns status code 400 with the message “No message available”. I have read probably 20 different forums/possible solutions now about how to send an image in python with POST but none of those solutions worked for this. I’ll include a link to pastebin so that you can view my code. https://pastebin.com/wUddgHsk Thank you so much for the help!

**Comment by aliubymov** [comment]
Hi @mattgoliver ! Thank's for your's feedback! I take a look at your code and it's a little bit wrong, I am correct it and now it's working, please see it here https://pastebin.com/35hRatBH
